### DISCUSSION TITLE
Install with Python 3.11.5

### DISCUSSION THREAD

**Discussion by huytradingz** [dodid/minitrade#1] [Q&A] [answered]
Hi there, I noticed that minitrade supports >= python 3.10 and <= 3.11. Is there any plan for minitrade to support 3.11.5 and later? If not, is there any workaround for MacOS? I am fairly new to python and version control for python, any help is appreciated! Thank you!

**Comment by dodid** [comment, marked as answer]
The restriction should have been removed in version 0.1.28. Please try!
